docs/development/tools/subkey.md

```diff
@@ -132,6 +132,21 @@ Secret phrase `favorite liar zebra assume hurt cage any damp inherit rescue dela
   Address (SS58): 13ToWeAsFe55Z7qGxwV8uJFch73aCEyHhhQb2hVAsspp4Xuo
 ```
 
+## Inserting Keys to Keystore
```

Comment:
Why did you put this as the first item in Subkey after installation? This is certainly not the primary use case since you need to actually generate the keys before you can insert them. Also nobody has any idea what a keystore is when they come to this page. This section should be at the end with a link to the keystore documentation.
```suggestion
## Inserting Keys into the Keystore
```